.github/workflows/verify_artifacts.py

```diff
@@ -0,0 +1,234 @@
+    "https://hyperledger.jfrog.io/hyperledger/besu-maven/org/hyperledger/besu/internal/config/VERSION/config-VERSION-javadoc.jar",
+    "https://hyperledger.jfrog.io/hyperledger/besu-maven/org/hyperledger/besu/besu-datatypes/VERSION/besu-datatypes-VERSION-sources.jar",
+    "https://hyperledger.jfrog.io/hyperledger/besu-maven/org/hyperledger/besu/internal/config/VERSION/config-VERSION-sources.jar",
+    "https://hyperledger.jfrog.io/hyperledger/besu-maven/org/hyperledger/besu/internal/besu/VERSION/besu-VERSION-sources.jar",
+    "https://hyperledger.jfrog.io/hyperledger/besu-maven/org/hyperledger/besu/besu-datatypes/VERSION/besu-datatypes-VERSION.jar",
+    "https://hyperledger.jfrog.io/hyperledger/besu-maven/org/hyperledger/besu/internal/besu/VERSION/besu-VERSION.jar",
+    "https://hyperledger.jfrog.io/hyperledger/besu-maven/org/hyperledger/besu/internal/config/VERSION/config-VERSION.jar",
+    "https://hyperledger.jfrog.io/hyperledger/besu-maven/org/hyperledger/besu/besu-datatypes/VERSION/besu-datatypes-VERSION.module",
+    "https://hyperledger.jfrog.io/hyperledger/besu-maven/org/hyperledger/besu/besu-datatypes/VERSION/besu-datatypes-VERSION.pom",
+    "https://hyperledger.jfrog.io/hyperledger/besu-maven/org/hyperledger/besu/internal/besu/VERSION/besu-VERSION.module",
+    "https://hyperledger.jfrog.io/hyperledger/besu-maven/org/hyperledger/besu/internal/config/VERSION/config-VERSION.module",
+    "https://hyperledger.jfrog.io/hyperledger/besu-maven/org/hyperledger/besu/internal/enclave/VERSION/enclave-VERSION-javadoc.jar",
+    "https://hyperledger.jfrog.io/hyperledger/besu-maven/org/hyperledger/besu/internal/config/VERSION/config-VERSION.pom",
+    "https://hyperledger.jfrog.io/hyperledger/besu-maven/org/hyperledger/besu/internal/besu/VERSION/besu-VERSION.pom",
+    "https://hyperledger.jfrog.io/hyperledger/besu-maven/org/hyperledger/besu/evm/VERSION/evm-VERSION-javadoc.jar",
+    "https://hyperledger.jfrog.io/hyperledger/besu-maven/org/hyperledger/besu/internal/nat/VERSION/nat-VERSION-javadoc.jar",
+    "https://hyperledger.jfrog.io/hyperledger/besu-maven/org/hyperledger/besu/internal/enclave/VERSION/enclave-VERSION-sources.jar",
+    "https://hyperledger.jfrog.io/hyperledger/besu-maven/org/hyperledger/besu/internal/enclave/VERSION/enclave-VERSION.jar",
+    "https://hyperledger.jfrog.io/hyperledger/besu-maven/org/hyperledger/besu/internal/nat/VERSION/nat-VERSION-sources.jar",
+    "https://hyperledger.jfrog.io/hyperledger/besu-maven/org/hyperledger/besu/evm/VERSION/evm-VERSION-sources.jar",
+    "https://hyperledger.jfrog.io/hyperledger/besu-maven/org/hyperledger/besu/internal/enclave/VERSION/enclave-VERSION.module",
+    "https://hyperledger.jfrog.io/hyperledger/besu-maven/org/hyperledger/besu/internal/nat/VERSION/nat-VERSION.jar",
+    "https://hyperledger.jfrog.io/hyperledger/besu-maven/org/hyperledger/besu/internal/enclave/VERSION/enclave-VERSION.pom",
```

Comment:
this is a very long list! are there any issues with having so many?

Also some of these are on the deprecated list so within 6 months we will remove - enclave, privacy

prob don't need sources.jar verified for all?

Reply:
Aye, that makes sense - no issues per say will take an extra couple of secs per artifact to check.

Do we have a core list of artifacts I can check against?